okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/UserAgentInterceptor.java

```diff
@@ -28,14 +31,31 @@
     private static final Pattern VALID_USER_AGENT = Pattern.compile("[A-Za-z0-9()\\-#;/.,_\\s]+");
     private final String userAgent;
 
-    private UserAgentInterceptor(String userAgent) {
+    private UserAgentInterceptor(String userAgent, Class<?> serviceClass) {
```

Comment:
We decided to get rid of service class version parsing in favor of passing explicit service+version at config time. Don't think we want to add this back.

Reply:
That's a different beast:
- In WC: version of the server/product
- Here: version of http-remoting and version of the (remote) API service class

Reply:
I see, ok.